framework/widget/src/main/java/org/apache/ofbiz/widget/model/ModelFormField.java

```diff
@@ -2738,25 +2745,30 @@ public ModelForm getModelGrid(Map<String, Object> context) {
      */
     public static class HiddenField extends FieldInfo {
         private final FlexibleStringExpander value;
+        private final FlexibleStringExpander disabled;
```

Comment:
It's not really common but you can imagine to have a `<input type="hidden" disabled />`, which is valid I think, according to the spec (this is why I added this field on `HiddenField`, to be complete). It renders as an invisible and not submitted field. Not very useful (event if I can imagine use cases where this could be handy), but since you can do it in HTML I thought we should support it.

Reply:
ok